src/dpq2/value.d

```diff
@@ -102,7 +102,7 @@ struct Value
         import std.conv: to;
         import std.variant;
 
-        return this.as!Variant.toString~"::"~oidType.to!string~"("~(format == ValueFormat.TEXT? "t" : "b")~")";
+        return this.as!(string)~"::"~oidType.to!string~"("~(format == ValueFormat.TEXT? "t" : "b")~")";
```

Comment:
Looks like that this is not an 100% equivalent replacement. It is need to pass `Variant` type to `as` template

Maybe this code will be better? Can you check it?
```d
this.as!Variant.to!string
```

Reply:
hm, bad. with the code you suggest the compile trace is the same as the original one.